projects/canopy/src/lib/side-nav/side-nav.component.scss

```diff
@@ -32,6 +32,7 @@
 
   &:visited {
     color: var(--side-nav-bar-link-color);
+    border-bottom: var(--border-width) solid var(--color-platinum);
```

Comment:
is it worth having `--color-platinum` as a var? something like `--side-nav-bar-link-border-color`

Reply:
Yeah, probably best actually. Will make that change